pkg/relay/ethereum/config.go

```diff
@@ -11,6 +15,40 @@ func (c ChainConfig) Build() (core.Chain, error) {
 	return NewChain(c)
 }
 
+func (c ChainConfig) Validate() error {
+	isEmpty := func(s string) bool {
+		return strings.TrimSpace(s) == ""
+	}
+
+	var errs []error
+	if isEmpty(c.ChainId) {
+		errs = append(errs, fmt.Errorf("config attribute \"chain_id\" is empty"))
+	}
+	if c.EthChainId <= 0 {
```

Comment:
@siburu `chainId = 0` is invalid? Seems that the popular networks do not use zero chain id, but each ethereum client implementation may allow it.

Reply:
I read some EIPs and discussions at Ethereum Magicians.
According to them, I'm not 100% confident, but `chainId = 0` doesn't mean a specific id but a wildcard?
If it's correct, I think that `chainId = 0` as `eth_chain_id` config attribute should be refused.

Reply:
@bluele Excuse me, but I changed my opinion.

The following EIP is not accepted, that is, `0` is a valid normal chain id at the moment.
https://eips.ethereum.org/EIPS/eip-3788

Reply:
@bluele I fixed this by changing the type of `eth_chain_id` from int64 to uint64.